src/trajectory.jl

```diff
@@ -334,7 +347,7 @@ Create an instance for the No-U-Turn sampling algorithm.
 function NUTS{S,C}(
     integrator::I,
     max_depth::Int=10,
-    Δ_max::F=1000.0
+    Δ_max::F=1000.0,
```

Comment:
@yebai We actually have a guide on always including "the trailing comma when working with expanded arrays, tuples or functions notation".